cartographer/mapping/internal/3d/scan_matching/interpolated_grid_test.cc

```diff
@@ -44,15 +44,15 @@ class InterpolatedGridTest : public ::testing::Test {
   }
 
   HybridGrid hybrid_grid_;
-  InterpolatedGrid interpolated_grid_;
+  InterpolatedProbabilityGrid interpolated_grid_;
```

Comment:
Is the intensity grid not intended to be used for some sort of scan matching or does it make sense to test its interpolation here, too?

Reply:
I don't think so. The interpolation is done in exactly the same way. How would adding a test with an intensity grid increase our confidence in the interpolation code? What has changed is the additional grid which is tested in `hybrid_grid_test.cc`.

Reply:
Right, both support `GetValue` so it's fine to test just one.